spec/views/posts/index.html.erb_spec.rb

```diff
@@ -1,5 +1,55 @@
 require 'rails_helper'
 
-RSpec.describe 'posts/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'posts/index.html.erb', type: :system do
+  describe 'index page' do
+    before do
+      # rubocop:disable Layout/LineLength
+      @first_user = User.create(name: 'Bob',
+                                image: 'https://icon-library.com/images/person-png-icon/person-png-icon-29.jpg', bio: 'Peruvian Chef.', posts_counter: 0)
+      @first_post = Post.create(author_id: @first_user.id, title: 'Hello', text: 'This is my first post',
+                                comments_counter: 0, likes_counter: 0)
+      # rubocop:enable Layout/LineLength
+    end
+
+    it 'show user profile picture' do
+      visit user_path(@first_user)
+      expect(page).to have_css("img[src*='https://icon-library.com/images/person-png-icon/person-png-icon-29.jpg']")
+    end
+
+    it 'show user name' do
+      visit user_path(@first_user)
+      expect(page).to have_content('Bob')
+    end
+
+    it 'see number of posts user has written' do
+      visit user_path(@first_user)
+      expect(page).to have_content('0')
+    end
+
+    it 'see post title' do
+      visit user_path(@first_user)
+      expect(page).to have_content('Hello')
+    end
```

Comment:
for these test cases there are no test cases for when :
`Tests that assert you can see a section for pagination if there are more posts than fit on the view.`

`Tests that assert you can see some of the post's body.`

please write and conduct these test cases again 💯